test/Website/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:1.0-runtime
+
```

Comment:
Need to add a "MAINTAINER" directive here. (e.g. https://hub.docker.com/r/prom/prometheus/~/dockerfile/)